modules/installation-mirroring-additional-images.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-restricted-networks-preparations.adoc
```

Comment:
I don't see this new module file actually included in the installing-restricted-networks-preparations.adoc assembly. So I don't see it at all in the preview.

Is this a module that is no longer needed?